events/apigw_test.go

```diff
@@ -177,6 +177,35 @@ func TestApiGatewayCustomAuthorizerResponseMarshaling(t *testing.T) {
 	assert.JSONEq(t, string(inputJSON), string(outputJSON))
 }
 
+func TestAPIGatewayV2CustomAuthorizerSimpleResponseMarshalling(t *testing.T) {
+	var tests = []struct {
+		desc string
+		in   APIGatewayV2CustomAuthorizerSimpleResponse
+		out  string
+	}{
+		{"defaults", APIGatewayV2CustomAuthorizerSimpleResponse{}, `{"isAuthorized":false}`},
+		{"without context", APIGatewayV2CustomAuthorizerSimpleResponse{IsAuthorized: true}, `{"isAuthorized":true}`},
+		{"context is nil", APIGatewayV2CustomAuthorizerSimpleResponse{Context: nil}, `{"isAuthorized":false}`},
+		{"context is empty", APIGatewayV2CustomAuthorizerSimpleResponse{Context: map[string]interface{}{}}, `{"isAuthorized":false}`},
+		{"context with basic types", APIGatewayV2CustomAuthorizerSimpleResponse{Context: map[string]interface{}{"string": "value", "bool": true, "number": 1234}}, `{"isAuthorized":false,"context":{"string":"value","bool":true,"number":1234}}`},
+		{"context with array", APIGatewayV2CustomAuthorizerSimpleResponse{Context: map[string]interface{}{"array": []string{"value1", "value2"}}}, `{"isAuthorized":false,"context":{"array":["value1","value2"]}}`},
+		{"context with map", APIGatewayV2CustomAuthorizerSimpleResponse{Context: map[string]interface{}{"map": map[string]string{"key": "value"}}}, `{"isAuthorized":false,"context":{"map":{"key":"value"}}}`},
+		{"context with nil value", APIGatewayV2CustomAuthorizerSimpleResponse{Context: map[string]interface{}{"nil": nil}}, `{"isAuthorized":false,"context":{"nil":null}}`},
+	}
+
+	for _, tt := range tests {
+		tt := tt
```

Comment:
IIRC you only need to do this if you are running things in parallel (which you actually could do here).

Reply:
In most cases I would agree. But without pinning, it can lead to subtle bugs in certain cases. Like described in the `loop pointer` (https://github.com/kyoh86/looppointer) linter used by `golangci-lint`. In this case you are absolutely right. I was just following the principal of pinning all the time like the linter suggests. I would leave it in if you don't mind.

Reply:
Yea. That's fine and makes sense.